src/feed/feed.service.ts

```
@@ -74,16 +75,21 @@ export class FeedService {
     job.status = status;
+  
     const updatedJob = await this.jobRepo.save(job);
-
-    // Notify the freelancer
-    await this.notificationsService.sendJobStatusNotification(job.freelancer.id, job.title, status);
-
+  
+    await this.notificationsService.sendJobStatusNotification(
+      job.freelancer.id,
+      job.title,
+      status as 'approved' | 'rejected' 
```

ESLint

[error] Replace `·` with `,`

(prettier/prettier)